prepared_statement_test.go

```diff
@@ -9,17 +9,18 @@ import (
 // TestPreparedStatement creates a basic prepared statement, inserting values
 // after the statement has been prepared
 func TestPreparedStatement(t *testing.T) {
-
 	runTests(t, dsn, func(dbt *DBTest) {
-		dbt.mustExec(createTableSQL)
```

Comment:
Any reason replace the createTableSQL and insertSQL below? They look like the same

Reply:
These query constants have now been modified for the tests in bindings_test.go